girder/api/v1/user.py

```diff
@@ -312,28 +319,24 @@ def generateTemporaryPassword(self, params):
         self.requireParams('email', params)
         email = params['email'].lower().strip()
 
-        users = self.model('user').find({'email': email})
+        user = self.model('user').findOne({'email': email})
 
-        if not users.count():
+        if not user:
             raise RestException('That email is not registered.')
-        for user in users:
```

Comment:
Was there a legitimate reason for this code for the possibility of multiple users with the same email?